quadriga/src/test/java/edu/asu/diging/quadriga/core/service/impl/CollectionManagerImplTest.java

```diff
@@ -44,11 +48,215 @@ public boolean matches(Collection arg0) {
                 return (arg0.getName().equals(name) && arg0.getDescription().equals(desc));
             }
         }))).thenReturn(savedCollection);
-        
+
         Collection collection = managerToTest.addCollection(name, desc);
         Assert.assertEquals(id, collection.getId());
         Assert.assertEquals(name, collection.getName());
         Assert.assertEquals(desc, collection.getDescription());
     }
 
+    @Test
+    public void test_findCollection_success() {
+        Collection collection = new Collection();
+        ObjectId objectId = new ObjectId();
+        collection.setId(objectId);
+        Mockito.when(collectionRepo.findBy_id(objectId)).thenReturn(collection);
+        Collection foundCollection = managerToTest.findCollection(objectId.toString());
+        Assert.assertEquals(collection.getId().toString(), foundCollection.getId().toString());
+    }
+
+    @Test
+    public void test_editCollection_success() {
+
+        String name = CollectionLiterals.COLLECTION_NAME;
```

Comment:
use the constant where it is needed.

Reply:
same everywhere else, there is no point in creating a new variable for it if it's only used once afterwards.

Reply:
I had added it as I am using it in two classes: EditCollectionControllerTest and CollectionManagerImplTest
Should I add these constants separately in these two classes?